spec/requests/api/v0/articles_spec.rb

```diff
@@ -261,6 +261,7 @@ def post_article(**params)
         tags = %w[meta discussion]
         post_article(body_markdown: "Yo ho ho", tags: tags)
         expect(response).to have_http_status(:unprocessable_entity)
+        expect(json_response["error"]).to be_present
```

Comment:
Should we add a test to check that an error is a string?

Reply:
Maybe it's overkill.

I see your point but AFAIK we tend not to test types around the code, so I didn't think of it

Reply:
Here I pointed this out because it was the reason for a fix.

Reply:
@lightalloy yeah, but the problem was also that the key was named `errors` instead of `error` :D